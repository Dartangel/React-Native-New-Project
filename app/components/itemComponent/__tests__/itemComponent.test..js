import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ItemComponent from '../index';

describe('ItemComponent', () => {
    test('ItemComponent snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<ItemComponent />);
        expect(snapshot).toMatchSnapshot();
    });
});