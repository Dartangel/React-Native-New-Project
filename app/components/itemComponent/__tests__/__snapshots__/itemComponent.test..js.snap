// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemComponent ItemComponent snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "borderRadius": 3,
      "flexDirection": "row",
      "height": 70,
      "marginBottom": 5,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.75,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 20,
          "fontWeight": "bold",
          "paddingLeft": 20,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.25,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "borderColor": "#08DBA5",
          "borderRadius": 50,
          "borderWidth": 4,
          "height": 50,
          "justifyContent": "center",
          "width": 50,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000000",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;
